docs/src/man/syntax.md

```diff
@@ -5,6 +5,7 @@ For supported Markdown syntax, see the [documentation for the Markdown standard
 
 ```@contents
 Pages = ["syntax.md"]
+MinDepth = 2
```

Comment:
Maybe it even makes sense to default `MinDepth = 2` when `lenght(Pages) == 1`?

Reply:
I am a little unsure about having one argument's default depending on the value of the other, since it could be surprising that changing `Pages` would change the depth behavior. On the other hand, I do see the utility. It would also remove any `h1`s further down the page though I believe?

Maybe a note in the documentation would be enough to push users to set `MinDepth = 2`?

Reply:
Yea, I was thinking that if you only have one entry in `Pages` your probably have it at the top of that same page and thus the H1 header is not needed. And multiple H1 headers on the same page is not so common. If you don't want this behavior you can of course override it. But perhaps we can do this by default at a later stage if it seems useful.